packages/app/src/KubernetesDashboardPage.tsx

```diff
@@ -528,6 +507,43 @@ const NodesTable = ({
   );
 };
 
+const K8sMetricTagValueSelect = ({
+  metricAttribute,
+  searchQuery,
+  setSearchQuery,
+  placeholder,
+  dropdownClosedWidth,
+  icon,
+}: {
+  metricAttribute: string;
+  searchQuery: string;
+  setSearchQuery: (query: string) => void;
+  placeholder: string;
+  dropdownClosedWidth: number;
+  icon: React.ReactNode;
+}) => {
+  return (
+    <MetricTagValueSelect
+      metricName="k8s.pod.cpu.utilization - Gauge"
+      metricAttribute={metricAttribute}
+      value={''}
```

Comment:
hmm

Reply:
yeah the ideal is that this is a multi-select that can parse the search query and represent that state here. totally doable as there is a query parser on the frontend - but I think that's too ambitious for now and instead this is just an abomination of UX for now...